recipes/phytop/src/Astral.py

```
@@ -0,0 +1,590 @@
+			#	node.img_style["draw_descendants"] = False
+				continue
+			try:
+				f1, f2, f3 = node.f1, node.f2, node.f3
+			except AttributeError: continue
+			f1, f2, f3 = map(float, [f1, f2, f3])
+			n = sum([f1, f2, f3])
+			if n == 0:
+				logger.warn('Zero in node: {}'.format(node.name))
+				continue
+			q1, q2, q3 = f1/n, f2/n, f3/n
+			coalescent_unit = node.dist
```

Ruff

Local variable `coalescent_unit` is assigned to but never used

Remove assignment to unused variable `coalescent_unit`

(F841)